core-web/libs/dotcms-scss/angular/dotcms-theme/components/_galleria.scss

```diff
@@ -0,0 +1,114 @@
+//TODO: Implement Galleria styles.
```

Comment:
so this file is unnecessary?

Reply:
it si now, but soon will work on that in a sub-tasks of the same feature.